Adfly/adfly_bot.py

```diff
@@ -0,0 +1,17 @@
+from selenium import webdriver
+import time
+
+def link():
+
+    driver.switch_to.window(driver.window_handles[1])
+    driver.get("http://raboninco.com/23079033/proves-boto-groc")
```

Comment:
Here you can put your adfly shortened link.

Reply:
http://lyksoomu.com/xl2U
http://lyksoomu.com/xl2V
http://lyksoomu.com/xl2W
http://lyksoomu.com/xl2X
http://lyksoomu.com/xl2Y
http://lyksoomu.com/xl2Z
http://lyksoomu.com/xl2a
http://lyksoomu.com/xl2b
http://lyksoomu.com/xl2c
http://lyksoomu.com/xl2d
http://lyksoomu.com/xl2e
http://lyksoomu.com/xl2f
http://lyksoomu.com/xl2g
http://lyksoomu.com/xl2h
http://lyksoomu.com/xl2i
http://lyksoomu.com/xl2j
http://lyksoomu.com/xl2k
http://lyksoomu.com/xl2l
http://lyksoomu.com/xl2m
http://lyksoomu.com/xl2n